app/controllers/hosts_controller.rb

```diff
@@ -479,6 +479,15 @@ def update_multiple_power_state
     redirect_back_or_to hosts_path
   end
 
+  def show_multiple_power_status
+    find_multiple_ajax
+    res = {}
+    @hosts.includes(:compute_resource).each do |host|
+      res[host.id] = host.supports_power_and_running? rescue false
```

Comment:
I agree it would be better to have all three states, the reason I have only two is to be consistent with what is under compute_resources/show, virtual machines tab. I could add third stated in a separate PR, this was just meant to add the on/off indicator to hosts page.

I do not generate html on client side, [look here](https://github.com/theforeman/foreman/pull/3459/files#diff-7eb571b855131f643240b7b93ddfd44dR25). Both on/off labels are already there, I just show/hide them based on what I get in response.